Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM alpine:3.3
-MAINTAINER jp@roemer.im
+FROM alpine:3.5
+
+# Components versions
+ENV GOLANG_VERSION 1.8
+ENV GOLANG_SRC_URL https://golang.org/dl/go$GOLANG_VERSION.src.tar.gz
+ENV GOLANG_SRC_SHA256 406865f587b44be7092f206d73fc1de252600b79b3cacc587b74b5ef5c623596
+
+# Branch name
+#ENV GOGS_VERSION v0.9.141
+ENV GOGS_VERSION develop
```

Comment:
>In the build-sh script I replaced the go get ... command by the git clone ... one (as underlined above) on purpose so that the desired Gogs version can be chosen.

You can choose with Docker Hub automated builds, this file is used for that. Not meant for manual build.